### DISCUSSION TITLE
App icon animating a loading in task bar

### DISCUSSION THREAD

**Discussion by EddyeBoy27** [rocksdanister/lively#1891] [Q&A]
I've configured Lively to stay in System Tray, but while my wallpaper is playing the Icon in task bar show an animating like a loading in same time of my wallpaper runs. All of that just when I startup my windows (the startup option is marked). I need to unmark the system tray option >> close app >> open app >> mark the system tray option again and the task bar icon app turn off and Lively stay in system tray.

Why ? It's a bug ?

**Comment by rocksdanister** [comment]
I am guessing its the mpv video player icon (violet circle.)

You can try switching to wmf player as workaround for now:
https://github.com/rocksdanister/lively/wiki/Video-Guide#windows-media-foundation-wmf

**Comment by EddyeBoy27** [comment]
Its the icon in task bar. (I just opened the app to show where this loading icon stay).
When I start my Windows, I have Lively in System Tray and in Task Bar (This one in Task Bar show a loading animate)

![image](https://github.com/rocksdanister/lively/assets/57309779/95c40635-c0c3-4a5a-827f-77b9370040e7)

**Reply by rocksdanister** [reply]
Did you by chance install both Store and installer version of Lively? only thing I could think of is something is launching the Lively again.

**Comment by EddyeBoy27** [comment]
No, I installed just the Microsoft Store version.
